onellm/utils/file_validator.py

```
@@ -0,0 +1,396 @@
+    # Image formats (for vision models)
+    ".png", ".jpg", ".jpeg", ".gif", ".webp", ".bmp", ".tiff", ".tif",
+    # Document formats
+    ".pdf", ".txt", ".json", ".jsonl", ".csv", ".tsv",
+    ".doc", ".docx", ".xls", ".xlsx", ".ppt", ".pptx",
+    # Code and data formats
+    ".py", ".js", ".ts", ".java", ".cpp", ".c", ".h",
+    ".xml", ".yaml", ".yml", ".toml", ".ini",
+    # Archive formats
+    ".zip", ".tar", ".gz", ".bz2", ".7z",
+    # Video formats (for future support)
+    ".mp4", ".avi", ".mov", ".mkv", ".wmv", ".flv",
```

Ruff (0.13.3)

Sets should not contain duplicate item `".mp4"`

Remove duplicate item

(B033)